### ISSUE TITLE
Wrong tag mapping from lst20 to UD

### ISSUE THREAD

**Issue by kepler** [PyThaiNLP/pythainlp#711]
## Description
With version 3.0.8, using the lst20_ud corpus for tagging (and perceptron) yields an `NG` tag that is not in the `TO_UD` map and breaks.

## Steps to reproduce
```python
from pythainlp import pos_tag
pos_tag(["ไม่"], corpus="lst20_ud")

KeyError: 'NG'
```

## Your environment
* PyThaiNLP version: 3.0.8
* Python version: 3.8.4
* Operating system and version (distro, 32/64-bit): ubuntu 22.04

**Comment by kepler** [comment]
I assume it's a matter of adding
```
    "NG": "PART",
```
to `TO_UD` in `tag/lst20.py`, considering Table 2.1 from https://arxiv.org/pdf/2008.05055.pdf.

**Comment by wannaphong** [comment]
@kepler Thank you for reporting. It fixed in #689 and it will be in PyThaiNLP v3.1.

**Comment by wannaphong** [comment]
@kepler Now, It fixed in PyThaiNLP v3.0.10. https://github.com/PyThaiNLP/pythainlp/releases/tag/v3.0.10

**Comment by kepler** [comment]
Cool! Thanks for the prompt release.
